visual-js/visual-snapshots/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@saucelabs/visual-snapshots",
+  "description": "CLI which generates Visual snapshots for a specified document",
+  "version": "0.13.0",
+  "main": "build/index.cjs",
+  "module": "build/index.js",
+  "types": "build/index.d.ts",
+  "license": "MIT",
+  "files": [
+    "build",
+    "README.md"
+  ],
+  "type": "module",
+  "engines": {
+    "node": "^16.13 || >=18"
+  },
+  "typeScriptVersion": "3.8.3",
```

Comment:
This seems to not correlate with the actual version of Typescript used in this project. Is this field necessary at all?